core/vm/contracts.go

```diff
@@ -139,7 +139,7 @@ var PrecompiledContractsPrague = PrecompiledContracts{
 
 var PrecompiledContractsBLS = PrecompiledContractsPrague
 
-var PrecompiledContractsVerkle = PrecompiledContractsPrague
+var PrecompiledContractsVerkle = PrecompiledContractsBerlin
```

Comment:
No KZG precompile?

Reply:
nope it's for a pre-cancun chain at the moment